pysoa/test/plugins/fixtures.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import (
```

Comment:
`plugins/pytest/fixtures.py`, please. Not `plugins/fixtures.py`. Also, below, `plugins/pytest/plans.py`, please. Not `plugins/plans.py`.

Reply:
so many directories